src/Dev/SapphireTest.php

```diff
@@ -334,6 +327,39 @@ protected function setUp(): void
         Email::config()->remove('bcc_all_emails_to');
     }
 
+    /**
+     * Sets the locale which unit-tests should be run in
+     */
+    protected function setI18nLocale(): void
```

Comment:
```suggestion
    private function setI18nLocale(): void
```
Unless you have reason to believe this is likely to be overridden, we should probably keep it private so we can remove or change it later without BC concerns

Reply:
Updated